packages/vite/src/node/plugins/terser.ts

```diff
@@ -81,12 +106,25 @@ export function terserPlugin(config: ResolvedConfig): Plugin {
       worker ||= makeWorker()
 
       const terserPath = loadTerserPath(config.root)
+      const nth =
```

Comment:
Looking at terser option types, it seems `mangle.properties` can also contain `nth` functions: https://github.com/vitejs/vite/blob/main/packages/vite/src/types/terser.d.ts#L118

I was going to propose a more generic function serialize / revive mechanism, but it looks like these are the only two places where functions may appear in all terser options.

@sapphi-red are you aware of other places where we use workers this way and may require handling functions?